rules/rules.libsonnet

```diff
@@ -2,4 +2,5 @@
 (import 'apps.libsonnet') +
 (import 'kube_scheduler.libsonnet') +
 (import 'node.libsonnet') +
-(import 'kubelet.libsonnet')
+(import 'kubelet.libsonnet') +
+(import 'windows.libsonnet')
```

Comment:
@povilasv this would add windows rules and dashboards into the default build - are you ok with that? Personally, I see no harm in including them but wanted to check with you.